static/locales/en-US.yaml

```diff
@@ -573,6 +573,7 @@ Settings:
     Manage Subscriptions: Manage Subscriptions
   Proxy Settings:
     Proxy Settings: Proxy
+    Proxy Warning: FreeTube doesn't have a built-in proxy but can connect to an external proxy, such as one running on your machine like Tor or an external one provided by a VPN. If enabled, make sure your proxy/Tor is set up, or FreeTube won't be able to fetch any data.
```

Comment:
I'm a bit concerned that the VPN part of the sentence will make people think they need to configure something in FreeTube when using a VPN.